runtime/compiler/aarch64/codegen/J9InstructionDelegate.hpp

```diff
@@ -48,6 +50,19 @@ class OMR_EXTENSIBLE InstructionDelegate : public J9::InstructionDelegate
 
    InstructionDelegate() {}
 
+public:
+
+   /**
+    * @brief Sets the return address to CallSnippet for Label target
+    * @param[in] cg : CodeGenerator
+    * @param[in] ins : instruction associated with CallSnippet
+    * @param[in] cursor : instruction cursor
+    */
+   static void encodeBranchToLabel(TR::CodeGenerator *cg, TR::ARM64ImmSymInstruction *ins, uint8_t *cursor)
+      {
+      ((TR::ARM64CallSnippet *)ins->getCallSnippet())->setCallRA(cursor + ARM64_INSTRUCTION_LENGTH);
```

Comment:
Because the InstructionDelegate class is an extensible class, the best practice is to not put any implementation into the header file because of the dependencies it might introduce.  Can you move this implementation into a J9InstructionDelegate.cpp file?

Reply:
Updated and added files based on the review comment.